app/customer.py

```diff
@@ -0,0 +1,46 @@
+import math
+

```

Comment:
There is no need to add a blank line between imports from the same group. Consider avoiding this in the future